index.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
-  "extends": "stylelint-config-standard-scss",
+  "extends": [
+    "stylelint-config-standard-scss",
+    "@stylistic/stylelint-config"
```

Comment:
Explanation from Stylelint docs:

> When we created these rules, pretty printers (like [Prettier](https://prettier.io/)) didn't exist. They now offer a better way to consistently format code, especially whitespace. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.

Basically they decided to remove support for those rules, and move them to the stylistic package for opt-in backwards compatibility. In our case, my opinion is that since we were relying on these rules before, we should continue using them even if Stylelint has decided to drop them.
In any case, this is up for discussion for sure. If we would instead like to keep aligned with Stylelint approach, then I think we would need to investigate wether to also use Prettier or not. For the time being, I think it makes sense to keep support for the same rules as originally planned, so projects won't be missing these features.

Reply:
Thanks. I agree, without Prettier they are needed.